terraform/distributed/README.md

```diff
@@ -184,6 +184,10 @@ Autoscaler infrastructure, with the exception of Cloud Scheduler, lives.
     gcloud alpha firestore databases create --region="${AUTOSCALER_APP_ENGINE_LOCATION}"
```

Comment:
The App Engine app is still required for use of Cloud Scheduler.  So, I suggest these commands be split into to two steps.

Step 8. Create a Google App Engine app, to enable the APIs for Cloud Scheduler and Firestore.

Step 9. Create database to store the state of the Autoscaler.  State can be stored in either Firestore or Cloud Spanner.